frontend/public/components/factory/table.tsx

```diff
@@ -428,6 +428,7 @@ export type TableProps = {
   loaded?: boolean;
   reduxID?: string;
   reduxIDs?: string[];
+  rowFilters?: any[];
```

Comment:
Not sure why this wasn't here, but `Table` does accept the `rowFilters`, so I think it should be added.